servicemap/schema.py

```diff
@@ -0,0 +1,52 @@
+from graphene import Field, ID, Int, List, NonNull, ObjectType
+from graphene_linked_events.schema import LocalisedObject, Response
+from servicemap.utils import api_client, format_response, json2obj, json_object_hook
+
+
+class ServiceUnitNode(ObjectType):
+    id = ID(required=True)
+    name = Field(LocalisedObject)
+
+
+class ServiceUnitNameListResponse(Response):
+    data = NonNull(List(NonNull(ServiceUnitNode)))
+
+
+class Query:
+    schools_and_kindergartens_list = Field(
+        ServiceUnitNameListResponse, page=Int(), page_size=Int()
+    )
+
+    @staticmethod
+    def normalize_service_unit_node(units: list):
+
+        return [
+            {
+                "id": "{data_source}:{place_id}".format(
```

Comment:
Is the "tprek" the only datasource here? I set the id to be "tprek:{ID}" so that it would be in same format as it is in the Kultus database, in LinkedEvents API and in rest of the system.